src/Redux/State/Board.spec.js

```diff
@@ -0,0 +1,33 @@
+import * as Board from './Board'
+
+describe('Redux :: Module :: Board', () => {
+  it('reduces to initial state', () => {
+    const state = Board.default()
+
+    expect(state).toEqual(Board.INITIAL_STATE)
+  })
+
+  it('reduces the add card action', () => {
+    const s1 = Board.default()
+    const action = Board.addCard()
+    const s2 = Board.default(s1, action)
+
+    expect(s2).not.toEqual(s1)
+
+    expect(s2.isLoading).toBe(true)
```

Comment:
Personal preference, but I always prefer to assert the whole state here instead of a single property.
Doing so you risk to miss potential bugs (especially if you go for a big refactoring of the component).

Reply:
So you want me to create an object and do something like this ?

```
const expectedObject = { isLoading: true, property2: "value" }

expect(s2).toBe(expectedObject)
```

Reply:
Sort of, as Hugo suggested you could do something like:
```
expect(s2).toBe({
  ...s1,
  isLoading: true,
})
```